mappings/net/minecraft/command/arguments/ArgumentTypes.mapping

```diff
@@ -17,3 +17,7 @@ CLASS net/minecraft/class_2316 net/minecraft/command/arguments/ArgumentTypes
 		FIELD field_10924 argClass Ljava/lang/Class;
 		FIELD field_10925 id Lnet/minecraft/class_2960;
 		FIELD field_10926 serializer Lnet/minecraft/class_2314;
+		METHOD <init> (Ljava/lang/Class;Lnet/minecraft/class_2314;Lnet/minecraft/class_2960;)V
+			ARG 1 argClass
```

Comment:
You use `argumentName` elsewhere, so this should be `argumentClass` to be consistent.